book/wildfires/sensors/wildfires-sensors-modis.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<p>When referring to libraries you can highlight them <code>hvplot</code> in the text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/acocac/environmental-ai-book/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>